.projenrc.ts

```diff
@@ -1543,7 +1543,7 @@ const integRunner = configureProject(
       '@aws-sdk/client-cloudformation',
     ],
     devDeps: [
-      'aws-cdk-lib',
+      'aws-cdk-lib@2.203.0',
```

Comment:
I think we only need this dependency to synthesize some assemblies with flags during tests, and only for the duration that the PR isn't released again.